src/onetasker/views.py

```diff
@@ -14,7 +14,7 @@
 
 from core import logic as core_logic, models, log
 from core.decorators import is_onetasker
-from core.email import get_email_content
+from core.email import get_email_body
 from submission.logic import handle_book_labels
 from core.setting_util import get_setting
```

Comment:
I know this is not in the PR, but this `core.settings_util` needs to be moved above in the imports.